types/semantic-ui-dropdown/global.d.ts

```diff
@@ -290,7 +290,7 @@ declare namespace SemanticUI {
             /**
              * @default 'auto'
              */
-            placeholder: 'auto' | 'value' | false;
+            placeholder: 'auto' | string | false;
```

Comment:
I am not sure if this is correct change... 'value' is also used in for example for `match` property.. Also if this is suppose to be string then the type needs to be `string | false` and not include auto. Refer https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes .. You will need sign off from owner of this library for this change.